addons/web_editor/static/src/scss/web_editor.common.scss

```diff
@@ -299,15 +299,17 @@ div.media_iframe_video {
     margin: 0 auto;
     text-align: center;
     position: relative;
-    overflow: hidden;
+    width: 100%;
+    padding-bottom: 56.25%; /* 16:9 aspect ratio */
```

Comment:
This change breaks the aspect ratio of videos in the "Image Gallery" snippet.

![image](https://github.com/user-attachments/assets/c8795b2b-9806-4cab-a5f7-19bdbd009ceb)